src/EventListener/PaymentPartialEventListener.php

```diff
@@ -29,6 +30,12 @@ public function __construct(
 
     public function __invoke(UnitsRefunded $unitRefunded): void
     {
+        if ($unitRefunded->comment() === RefundOriginInterface::MOLLIE_WEBHOOK) {
```

Comment:
This is extremely tricky because we rely on the key setting in the comment. I wonder if we are unable to check in some way whether a given refund has already taken place 🤔